flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/TaskExecutorResourceUtilsTest.java

```diff
@@ -78,8 +78,9 @@ private static Configuration setAllRequiredOptionsExceptOne(ConfigOption<?> opti
 		//noinspection unchecked
 		TaskExecutorResourceUtils.CONFIG_OPTIONS
 			.stream()
-			.filter(option -> option.equals(TaskManagerOptions.CPU_CORES))
-			.filter(option -> option.equals(optionToNotSet))
+			.filter(option -> !option.equals(TaskManagerOptions.CPU_CORES))
+			.filter(ConfigOption::hasDefaultValue)
```

Comment:
I guess this should go to the call site where we filter out `CONFIG_OPTIONS` which have a default value.